apps/designer/app/designer/features/style-panel/sections/flex-child/flex-child.tsx

```diff
@@ -38,6 +38,11 @@ const FlexChildSectionAlign = (props: RenderCategoryProps) => {
       <PropertyName
         property={sectionStyle.alignSelf?.styleConfig.property}
         label="Align"
+        onReset={() => {
+          sectionStyle.alignSelf.setProperty(
+            sectionStyle.alignSelf?.styleConfig.property
+          )({ type: "unset", value: "" });
```

Comment:
why do we need a value if we are unsettling?

Reply:
Because it was defined in types this way and some logic relied on "value" prop.
cc @istarkov

Reply:
lol looks like I defined it 2month ago, no idea what I was thinking since we are not using that value when we delete the property

Reply:
@istarkov if you don't see why we need that value, I would remove it

Reply:
I think value was added to just remove errors on `String(style.value)` imo now can be safely removed as I think I changed all the places where it was used